docs/advanced/model_examples.rst

```diff
@@ -211,6 +211,9 @@ Now, if you do not want to show nor take into account the model uncertainties, s
 .. note::
     To plot the uncertainty of the model as a hashed area, as done in :func:`plot_model() <plothist.plotters.plot_model>`, you can use the standalone function :func:`plot_hist_uncertainties() <plothist.plotters.plot_hist_uncertainties>`.
 
+.. note::
+    In the two examples above, the bin width is hardcoded in the `ylabel`. With boost-histogram, it is possible to get the bin width from the histogram using `h.axes[0].widths[0]`.
```

Comment:
With boost-histogram, it is possible to get the bin width from the histogram using `h.axes[0].widths[0]` 

-> 

For a 1D histogram with a regular binning, it is possible to get the bin width from the `boost_histogram.Histogram` object using `h.axes[0].widths[0]`